app/views/notifications_mailer/match_initiation_for_manual_activation.haml

```diff
@@ -0,0 +1,17 @@
+Hello #{@contact.full_name},
```

Comment:
@awisema let's review this in context together, I think the wording needs be updated for the set-asides.